2.13/tests/test/test.js

```diff
@@ -675,3 +747,71 @@ function validateJsonSchema(body, schema) {
     }
     return '';
 }
+
+function validateCatalogSchema(tempBody, schema_type, action) {
+    preparedRequest()
+        .get('/v2/catalog')
+        .set('X-Broker-API-Version', apiVersion)
+        .auth(config.user, config.password)
+        .expect(200)
+        .expect('Content-Type', /json/)
+        .end(function(err, res){
+            if (err) {
+                var message = "Get catalog result failed: " + err;
+                return message;
+            }
+            var catalog = JSON.parse(JSON.stringify(res.body));
+            var schemaCheckResults = parametersSchemaCheck(catalog, tempBody.service_id, tempBody.plan_id, schema_type, action, tempBody.parameters);
+            if (schemaCheckResults != '') {
+                var message = "Validate schema parameters failed!";
+                return message;
+            }
+        })
+    return '';
+}
+
+function containsKeyValue(obj, key, value) {

```

Comment:
The name looks like a function which would return Boolean. What about renaming it to `findWhichContains()`?